wsmaster/che-core-api-factory-azure-devops/src/main/java/org/eclipse/che/api/factory/server/azure/devops/AzureDevOpsUserDataFetcher.java

```diff
@@ -57,10 +60,34 @@ protected GitUserData fetchGitUserDataWithOAuthToken(String token)
   protected GitUserData fetchGitUserDataWithPersonalAccessToken(
       PersonalAccessToken personalAccessToken)
       throws ScmItemNotFoundException, ScmCommunicationException, ScmBadRequestException {
-    AzureDevOpsUser user =
-        azureDevOpsApiClient.getUserWithPAT(
-            personalAccessToken.getToken(), personalAccessToken.getScmOrganization());
-    return new GitUserData(user.getDisplayName(), user.getEmailAddress());
+    if (personalAccessToken.getScmProviderUrl().equals("https://dev.azure.com")) {
```

Comment:
We have `AzureDevOps` class for contants

Reply:
done